# On the advice of the upstream author, patch cabal file to depend on polyparse
# 1.3 (the latest) instead of 1.2.
Index: haxml-1.19.7/HaXml.cabal
===================================================================
--- haxml-1.19.7.orig/HaXml.cabal
+++ haxml-1.19.7/HaXml.cabal
@@ -52,7 +52,7 @@
         Text.XML.HaXml.Xtract.Lex,
         Text.XML.HaXml.Xtract.Parse
   hs-source-dirs: src
-  build-depends: base, haskell98, polyparse==1.2, filepath
+  build-depends: base, haskell98, polyparse==1.3, filepath
   if flag(splitBase)
     build-depends: base >= 3, pretty, random, containers
   else
